pkg/kubelet/pleg/generic.go

```diff
@@ -192,6 +192,12 @@ func (g *GenericPLEG) relist() {
 		metrics.PLEGRelistInterval.Observe(metrics.SinceInMicroseconds(lastRelistTime))
 	}
 
+	_, err := g.runtime.Status()
+	if err != nil {
+		klog.Errorf("Container runtime sanity check failed: %v", err)
```

Comment:
Let's say the container runtime is dead, and PLEG returned here with logged this line of error. Then what will happen next? 

What will be the status of Pods? And how user knows the container runtime is dead? (user will never check log unless sth really bad happens)

Reply:
Container runtime dead not represent container dead, container will not be impacted. We return here, 3 minutes later, the node status will be not ready which we get from apiserver.